### DISCUSSION TITLE
Migration from terraform v1.9.2 to OpenTofu Latest

### DISCUSSION THREAD

**Discussion by rishiraja18** [opentofu/opentofu#2036] [Q&A]
I want to migrate the AWS resources from Terraform v1.9.2 to OpenTofu latest version. could someone suggest the steps if you have done and succeeded because OpenTofu yet to provide migration guide for the migration from terraform version greater than v1.8.2.
Thanks in advance

**Comment by cam72cam** [comment]
Hi @rishiraja18, if you are not using any of the new features in TF 1.9.x, the migration should be straightforward.  If you are using the new TF features in 1.9.x, you will need to wait for OpenTofu 1.9.0 to migrate.

**Reply by cam72cam** [reply]
As always, backup your state file and try the migration on a development/testing environment first.

**Reply by rishiraja18** [reply]
@cam72cam  Thanks I don't think we use any features in 1.9.2 will try the upgrade in the test env. Thanks again for your response

**Reply by ghost** [reply]
@cam72cam I believe we should update our migration guides to include 1.9.
